skycoin/skywire
# Improvements for the manager UI

Did you run `make format && make check`?
The go code was not changed. `npm run lint` and `npm run build` were used.

Fixes #923

 Changes:	
- The manager was adapted to work with the new health info format.
- The option for making a transport persistent was fixed.
- The option for configuring the `transport.public_autoconnect` setting was added to the UI.

How to test this PR:
- For checking the new node health info: check the right bar in the visor details page, the Health Info section should work well. Also, in the visor list, the color dots at the left should work well.
- For the persistent transport option: press the star at the left of a transport in the transport list, in the visor details page.
- For the `transport.public_autoconnect` setting: open the visor details page and use the option in the Transports Info section, in the bar at the right.

NOTE: while testing with the standard test enviroment, it was posible to add and remove persistent transports in the `0348c...` visor (visorB), but there was a problem with the `024ec...` and `031b8...` visors (visorA and visorC). More specifically, the `GET  /api/visors/{pk}/persistent-transports` API endpoint returns `{"error":"rpc: can't find method app-visor.GetPersistentTransports"}`. Not sure about the cause of the error, but may need atention.
